WooCommerce/Classes/ViewRelated/Orders/Order Creation/EditableOrderViewModel.swift

```diff
@@ -168,13 +168,18 @@ final class EditableOrderViewModel: ObservableObject {
     /// View model for the product list
     ///
     lazy var addProductViewModel = {
-        ProductSelectorViewModel(siteID: siteID, purchasableItemsOnly: true, storageManager: storageManager, stores: stores) { [weak self] product in
-            guard let self = self else { return }
-            self.addProductToOrder(product)
-        } onVariationSelected: { [weak self] variation, parentProduct in
-            guard let self = self else { return }
-            self.addProductVariationToOrder(variation, parent: parentProduct)
-        }
+        ProductSelectorViewModel(
+            siteID: siteID,
+            purchasableItemsOnly: true,
+            storageManager: storageManager,
+            stores: stores,
+            supportsMultipleSelections: ServiceLocator.featureFlagService.isFeatureFlagEnabled(.productMultiSelectionM1)) { [weak self] product in
```

Comment:
Nit: For initializers that include more than one closure, it's best to keep their names rather than using trailing closures to ensure readability:

```suggestion
            supportsMultipleSelections: ServiceLocator.featureFlagService.isFeatureFlagEnabled(.productMultiSelectionM1), onProductSelected: { [weak self] product in
```

Reply:
Totally agree. I changed this on other places, but I may have missed this one. Changed on https://github.com/woocommerce/woocommerce-ios/pull/8935/commits/79598230157639942abf6482f6baf25d742ca670